app/HutchAgent/hutchagent/query.py

```diff
@@ -15,13 +17,12 @@
     not_,
     or_,
     select,
-    table,
 )
 from typing import Any
-
-import hutchagent.config as hutch_config
+from hutchagent.db_manager import SyncDBManager
 from hutchagent.entities import ConditionOccurrence, Measurement, Observation, Person
 
+dotenv.load_dotenv()
```

Comment:
❓do you really need to do this in multiple modules, not just once for the entrypoint?

(this question is based on no prior knowledge)

Reply:
I got an error when I didn't so yes. Though I could try putting the statement once at the top of main.py, above all the other import statements so that the first thing done is loading the environment variables. It's just ugly from a stylistic point of view (i.e. doing app logic in the middle of imports).